### PR TITLE
benckmarks: integrated the two README.md 

### PR DESCRIPTION
**Pull request by RiyaTyagi**
The readme file present at "benchmarks/src/jmh/java/io/grpc/benchmarks/netty/README.md" is integrated with readme at "benchmarks/README.md".

### CODE REVIEW

File: benchmarks/README.md [hunk 1]
```diff
@@ -17,6 +17,39 @@ You can now find the client and the server executables in `benchmarks/build/inst
 
 The `C++` counterpart can be found at https://github.com/grpc/grpc/tree/master/test/cpp/qps
 
+This [netty](https://github.com/RiyaTyagi/grpc-java/tree/master/benchmarks/src/jmh/java/io/grpc/benchmarks/netty) directory contains the standard benchmarks used to assess the performance of GRPC. Since these benchmarks run on localhost over loopback, the performance of the underlying network is considerably different to real networks and their behavior. To address this issue we recommend the use of a network emulator to make loopback behave more like a real network. To this end the benchmark code looks for a loopback interface with 'benchmark' in its name and attempts to use the address bound to that interface when creating the client and server. If it cannot find such an interface it
```
Comment (ejona86) [inline benchmarks/README.md]: Please word-wrap this paragraph. And other paragraphs (no need for code snippets)

Comment (ejona86) [inline benchmarks/README.md]: The URL is to the wrong project. It is probably better to just say the path and not worry about making a link.

File: benchmarks/README.md [hunk 2]
```diff
@@ -17,6 +17,39 @@ You can now find the client and the server executables in `benchmarks/build/inst
 
 The `C++` counterpart can be found at https://github.com/grpc/grpc/tree/master/test/cpp/qps
```
Comment (ejona86) [inline benchmarks/README.md]: Add a header here? Maybe `## Netty Benchmarks`

**Comment by ejona86** [comment]
This PR is to address #594

File: benchmarks/README.md (hunk 2 above)
Comment (RiyaTyagi) [reply benchmarks/README.md]: I will do all the required changes to the readme. Sorry for the wrong URL.

**Comment by ejona86** [comment]
@RiyaTyagi, the paragraph's aren't word-wrapped and some of the content was removed.

**Pull request merged**